client/signup/main.jsx

```diff
@@ -725,30 +719,45 @@ class Signup extends React.Component {
 		const showProgressIndicator = 'pressable-nux' === this.props.flowName ? false : true;
 
 		return (
-			<div className={ `signup is-${ kebabCase( this.props.flowName ) }` }>
-				<DocumentHead title={ this.props.pageTitle } />
-				{ ! isWPForTeamsFlow( this.props.flowName ) && (
-					<SignupHeader
-						positionInFlow={ this.getPositionInFlow() }
-						flowLength={ this.getFlowLength() }
-						flowName={ this.props.flowName }
-						showProgressIndicator={ showProgressIndicator }
-						shouldShowLoadingScreen={ this.state.shouldShowLoadingScreen }
-						isReskinned={ this.props.isReskinned }
-					/>
-				) }
-				<div className="signup__steps">{ this.renderCurrentStep() }</div>
-				{ ! this.state.shouldShowLoadingScreen && this.props.isSitePreviewVisible && (
-					<SiteMockups stepName={ this.props.stepName } />
-				) }
-				{ this.state.bearerToken && (
-					<WpcomLoginForm
-						authorization={ 'Bearer ' + this.state.bearerToken }
-						log={ this.state.username }
-						redirectTo={ this.state.redirectTo }
-					/>
-				) }
-			</div>
+			<ProvideExperimentData name="refined_reskin_v1">
+				{ ( isLoading, experimentAssignment ) => {
+					if ( isLoading ) {
+						return null;
+					}
+
+					const isReskinned =
+						'onboarding' === this.props.flowName &&
+						'treatment' === experimentAssignment?.variationName;
+					! isLoading && ! isReskinned && document.body.classList.remove( 'is-white-signup' );
+
```

Comment:
```suggestion
			<ProvideExperimentData 
			  name="refined_reskin_v1"
			  options={{ isEligible: 'onboarding' === this.props.flowName }}
			  >
				{ ( isLoading, experimentAssignment ) => {
					if ( isLoading ) {
						return null;
					}

					const isReskinned = 'treatment' === experimentAssignment?.variationName;
					! isLoading && ! isReskinned && document.body.classList.remove( 'is-white-signup' );

```

`isEligible` is available now and I recommend using it to improve analysis.

Reply:
Thanks @jessie-ross , I've implemented the suggested changes.

Reply:
lgtm